### PR TITLE
Preserves position of SplitPane from redux store & moves appComponent out of folder

### PR DESCRIPTION
**Pull request by anmolarora1**

### CODE REVIEW

File: src/components/AppComponent.js [hunk 1]
```diff
@@ -36,13 +35,15 @@ import {
   EM_TOKEN,
 } from '../constants'
 
+import { updateSplitPosition } from '../action-creators/splitView'
```
Comment (raineorshine) [inline src/components/AppComponent.js]: I believe this should be:

```js
import { updateSplitPosition } from '../action-creators/updateSplitPosition'
```

File: src/util/initialState.js [hunk 2]
```diff
@@ -60,6 +60,7 @@ export const initialState = () => {
     schemaVersion: SCHEMA_LATEST,
     showHiddenThoughts: false,
     showSidebar: false,
+    splitPosition: localStorage.getItem('splitPosition') ? JSON.parse(localStorage.getItem('splitPosition')) : 0,
```
Comment (raineorshine) [inline src/util/initialState.js]: This is a bit cleaner and does not involve two calls to `getItem`:

```js
splitPosition: JSON.parse(localStorage.getItem('splitPosition') || '0')
```

Comment (anmolarora1) [reply src/util/initialState.js]: Good catch. Will update it

File: src/components/AppComponent.js (hunk 1 above)
Comment (anmolarora1) [reply src/components/AppComponent.js]: Agreed

**Pull request merged**